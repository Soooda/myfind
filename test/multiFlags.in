--name="myfind.py"
.
--name="ok.py"