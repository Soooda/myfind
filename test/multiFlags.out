Usage: myfind [--regex=pattern | --name=filename] directory [command]